### ISSUE TITLE
Paste without formatting should/could not be applied to copying files?

### ISSUE THREAD

**Issue by martinengelke** [p0deje/Maccy#962]
### Before Submitting Your Bug Report

- [x] I have verified that there isn't already an issue reporting the same bug to prevent duplication.
- [x] I have seen the [FAQ](https://github.com/p0deje/Maccy?tab=readme-ov-file#faq).

### Maccy Version (see 'About' window)

2.0.0 (47)

### macOS Version

14.7.1

### Maccy Settings

```Shell
{
    "KeyboardShortcuts_delete" = "{\\"carbonModifiers\\":256,\\"carbonKeyCode\\":51}";
    "KeyboardShortcuts_pin" = "{\\"carbonModifiers\\":256,\\"carbonKeyCode\\":35}";
    "KeyboardShortcuts_popup" = "{\\"carbonKeyCode\\":9,\\"carbonModifiers\\":6912}";
    "LaunchAtLogin__hasMigrated" = 1;
    "NSStatusItem Visible Item-0" = 1;
    "NSStatusItem Visible Item-1" = 0;
    "NSWindow Frame com.sindresorhus.Settings.FrameAutosaveName" = "501 399 510 435 0 0 1512 950 ";
    SUAutomaticallyUpdate = 0;
    SUEnableAutomaticChecks = 0;
    SUHasLaunchedBefore = 1;
    SUSendProfileInfo = 0;
    enabledPasteboardTypes =     (
        "public.html",
        "public.png",
        "public.utf8-plain-text",
        "public.tiff",
        "public.rtf",
        "public.file-url"
    );
    highlightMatch = underline;
    ignoredPasteboardTypes =     (
        "Pasteboard generator type",
        "com.typeit4me.clipping",
        "net.antelle.keeweb",
        "com.agilebits.onepassword",
        "de.petermaurer.TransientPasteboardType"
    );
    imageMaxHeight = 40;
    menuIcon = paperclip;
    migrations =     {
        "2024-07-01-version-2" = 1;
    };
    numberOfUsages = 279;
    pasteByDefault = 1;
    pinTo = top;
    popupPosition = lastPosition;
    previewDelay = 20000;
    removeFormattingByDefault = 0;
    searchMode = mixed;
    showFooter = 1;
    showInStatusBar = 1;
    showRecentCopyInMenuBar = 1;
    showSearch = 1;
    showTitle = 1;
    sortBy = firstCopiedAt;
    suppressClearAlert = 1;
    windowPosition = "[0.5076058201058201,0.9347368421052632]";
    windowSize = "[955,497]";
}
```

### Description

If I select "Paste without formatting" as standard, which is my preferred choice, I am not able to copy and paste Files in e.g. Finder. The paste will then just fill in the pure filename in maccy itself it displays:

<img width="945" alt="Bildschirmfoto 2024-11-24 um 12 56 12" src="https://github.com/user-attachments/assets/bd41c17f-6f3e-4cd4-ad39-d8c0036c82ec">

In text fields etc. I can then paste it like: 
**debugMenuItems.js**

The rest is just cut off. 

If I disable "Paste without formatting" it will bring up the file itself, which I prefer in case I selected a file. 

If I copy the filename by clicking on the file I can see, where the problem is located. It will also copy the complete file path with file:/// in the beginning. So there is no difference between selecting a file or selecting the filename by first click on the file, which maybe should be the case? 

Tldr;
I would like to be able to have "paste without formatting" as standard configuration in combination with copying files without extra keystrokes.
